modules/nw-metallb-configure-address-pool.adoc

```diff
@@ -65,4 +65,4 @@ Spec:
 Events:         <none>
 ----
 
-Confirm that the address pool name, such as `doc-example`, and the IP address ranges appear in the output.
+Confirm that the address pool name, such as `doc-example`, and the IP address ranges shown in the output.
```

Comment:
```suggestion
Confirm that the address pool name, such as `doc-example`, and the IP address ranges are present in the output.
```

- Up to you
- Is this a verification step? If yes then `.Verification` can be used -- up to you

Reply:
:heavy_check_mark: